MailCore/Cache/MailboxManager/MailboxManageable.swift

```diff
@@ -27,7 +28,8 @@ public typealias MailboxManageable = MailboxManagerCalendareable
     & MailboxManagerMailboxable
     & MailboxManagerMessageable
     & MailboxManagerSearchable
-    & RealmAccessible
+    & RealmConfigurable
```

Comment:
Notice it no longer conforms to `RealmAccessible` (.getRealm), but only the realm configuration `RealmConfigurable` and `Transactionable`